_posts/pandera-dataframe-validation.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: single
+title: "Pandera: A Statistical Data Validation Toolkit for Pandas"
+excerpt: "A flexible and expressive toolkit for performing statistical validation checks on pandas data structures."
+author: "Niels Bantilan"
+permalink: /blog/pandera-python-pandas-dataframe-validation
+header:
+    overlay_image: images/pandas.jpg
+    overlay_filter: 0.6
+categories:
+  - blog-post
+  - pandas
+  - data-validation
+  - dataframes
+---
+
+<span style='font-size: 15px;'>
+    Photo credit:
+    <a href="https://www.flickr.com/photos/nationalzoo/5371290900/in/photostream/"
+       target="_blank">
+    Ann Batdorf, Smithsonian's National Zoo</a>
+</span>
+
+Modern data engineering and analysis workflows will often involve using data
+manipulation libraries, which, in the Python universe, would be tools like
+[pandas](https://pandas.pydata.org/). One problem you may have encountered with
+this powerful data manipulation tool is that the dataframe can be an opaque
+object that's hard to reason about in terms of its contents, data types, and
+other properties.
+
+One tool that may help you with is problem is
+[pandera](https://pandera.readthedocs.io/en/latest/index.html), which was
+accepted by pyOpenSci as part of its ecosystem of packages on September 2019.
+Pandera provides a flexible and expressive data validation toolkit that helps
+users make statistical assertions about pandas data structures.
+
+## A Statistical Data Validation Toolkit for Pandas
+
+<img src="https://pandera.readthedocs.io/en/latest/_images/pandera-logo.svg"
+width="250px">
+
+Let's say you're analyzing some data for some insights in the context of a
+mission-critical project, where it’s vital to ensure the quality of the
+datasets that you're looking at.
```

Comment:
If you added another few sentences like

"you want to ensure that some columns have the correct data type, or that certain statistical properties of each column exist in the data. Pandera allows you to validate a DataFrame to ensure that these conditions are met. It allows you to spend less time worrying about the correctness of a DataFrame's data so you can make the right assumptions in analyzing it."

Then it could transition nicely.

Reply:
thanks for the feedback! when I moved some paragraphs around I didn't notice that it killed the flow a bit. adding your suggestion.